File: app/src/main/java/com/example/bmiapp/FeedSuggestionGenerator.java
package com.example.bmiapp;

public class FeedSuggestionGenerator {

    public String generateFeed(float bmi) {
        if (bmi < 16) {
            return "Mięso, warzywa, kasze, ryż";
        } else if (bmi < 18.5) {
            return "Mięso, warzywa, kasze";
        } else if (bmi < 25) {
            return "Warzywa, owoce, wędliny";
        } else if (bmi < 30) {
            return "Dużo wody, warzywa, produkty mało przetworzone, aktywność fizyczna";
        } else {
            return "Dużo wody, warzywa, produkty mało przetworzone, zacznij się ruszać(najlepiej rowerek stacjonarny), nie podjadaj wieczorem";
        }
    }
}

File: app/src/main/java/com/example/myapplication/FeedSuggestionGenerator.kt
package com.example.myapplication


class FeedSuggestionGenerator {
    fun generateFeed(bmi: Float): String {
        return when {
            bmi < 16 -> "Mięso, warzywa, kasze, ryż"
            bmi < 18.5 -> "Mięso, warzywa, kasze"
            bmi < 25 -> "Warzywa, owoce, wędliny"
            bmi < 30 -> "Dużo wody, warzywa, produkty mało przetworzone, aktywność fizyczna"
            else -> "Dużo wody, dieta niskokaloryczna, regularne badania"
        }
    }
}
